examples/graphql-field-interceptors/graphql_field_interceptors.md

```diff
@@ -0,0 +1,25 @@
+# GraphQL service - Field Interceptors
+
+The GraphQL resolver functions allow adding interceptors for GraphQL requests to execute custom logic. An interceptor can be defined using a `readonly` class that includes the `graphql:Interceptor` type. The interceptor class must implement the `execute` remote method, which is defined in the `graphql:Interceptor` service object type. It can be passed as a single interceptor or an array of interceptors using the `interceptors` field in the `graphql:ResourceConfig` annotation. The provided interceptors will be executed using the _onion principle_. Use the field interceptors to execute custom logic before and after executing a `resource` or a `remote` method that needs to be separated from the business logic.
+
+>**Note:** A resolver can have zero or more interceptors.
+
+::: code graphql_field_interceptors.bal :::
+
+Run the service by executing the following command.
```

Comment:
```suggestion
Run the service by executing the following command.
```
Shall we add the instruction steps in numbered bullet points?

Reply:
This is the same structure we followed for other BBEs. If we do this change, it will break the consistency.